sites/public/pages/eligibility/age.scss

```diff
@@ -0,0 +1,7 @@
+.age-field input {
+  max-width: 3.5rem;
```

Comment:
I think want to use tailwind classes, to stay consistent with styling in the rest of the site.

It's not max-width, but to set the width it'd be something like
`@apply w-14` I think.
https://tailwindcss.com/docs/width for the class
and
https://tailwindcss.com/docs/functions-and-directives#apply for the `@apply`

Reply:
I'm actually unable to apply core tailwind classes in the sites directory. I'm only able to apply the styles that are exported from the ui-components tailwind config: https://github.com/bloom-housing/bloom/blob/master/ui-components/tailwind.config.js

Maybe we should keep this as is for now and revisit at a later time?

Reply:
works for me, but can you leave a TODO explaining that and why this is here?

Also as discussed in chat, I think long term we might not want to be styling the `input` from outside the <Field> component, but this works for now.

Reply:
Done.